src/tests/ftest/util/ior_utils.py

```diff
@@ -139,6 +140,83 @@ def thread_run_ior(thread_queue, job_id, test, manager, log, hosts, path, slots,
         thread_queue.put(thread_result)
 
 
+def write_data(test, container, namespace='/run/ior_write/*', **ior_run_params):
+    """Write data to the container/dfuse using ior.
+
+    Simple method for test classes to use to write data with ior. While not required, this is setup
+    by default to pull in ior parameters from the test yaml using a format similar to:
+
+        ior: &ior_base
+          api: DFS
+          transfer_size: 512K
+          block_size: 1G
+          ppn: 2
+
+        ior_write:
+          <<: *ior_base
+          flags: "-k -v -w -W -G 1"
+
+        ior_read:
+          <<: *ior_base
+          flags: "-v -r -R -G 1"
+
+    Args:
+        test (Test): avocado Test object
+        container (TestContainer): the container to populate
+        namespace (str, optional): path to ior yaml parameters. Defaults to '/run/ior_write/*'.
+        ior_run_params (dict): optional params for the Ior.run() command, like ppn, dfuse, etc.
+
+    Returns:
+        Ior: the Ior object used to populate the container
+    """
+    job_manager = get_job_manager(test, subprocess=False, timeout=60)
+    ior = get_ior(test, job_manager, test.hostlist_clients, test.workdir, None, namespace)
+
+    if 'processes' not in ior_run_params:
+        ior_run_params['processes'] = test.params.get('processes', namespace, None)
+    elif 'ppn' not in ior_run_params:
+        ior_run_params['ppn'] = test.params.get('ppn', namespace, 1)
```

Comment:
So default processes to None and ppn to 1? Should they both just be None to avoid confusion?